ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss

```diff
@@ -60,6 +65,22 @@
     min-width: fit-content;
 }
 
+@media screen and (max-width: 1400px) and (min-width: $desktop-breakpoint) {
```

Comment:
Added styles for a `non-standard` screen resolution breakpoint in order to fit a new link into the already existing footer.